src/validation/validationSchema.ts

```diff
@@ -0,0 +1,152 @@
+/*!
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: MIT
+ */
+
+export default {
+    Common: {
+        Next: true,
+        End: true,
```

Comment:
Choice, Succeed, and Fail don't have End/Next properties, do we want to keep these in Common?

Reply:
Good catch. I followed the documentation which incorrectly states that Choice has common issues. I've reported a bug for it. I'll add 'hasCommonFields: false` flag to the `Choice` state.